Frends.Csv.Tests/Tests.cs

```diff
@@ -215,39 +215,50 @@ public void TestWriteFromListTable()
                 new List<object>() {100, "Dilantin", "Melanie", date}
```

Comment:
https://github.com/FrendsPlatform/Frends.Csv/blob/e1e43f24cf9843d6caa80cb1af4eca3a6d877aab/Frends.Csv.Tests/Tests.cs#L29
Please flip the expected value and actual value around. First parameter should be the expected value. This concerns all AreEqual-checks.
https://github.com/FrendsPlatform/Frends.Csv/blob/e1e43f24cf9843d6caa80cb1af4eca3a6d877aab/Frends.Csv.Tests/Tests.cs#L138
Please use `AreEqual`.
https://github.com/FrendsPlatform/Frends.Csv/blob/e1e43f24cf9843d6caa80cb1af4eca3a6d877aab/Frends.Csv.Tests/Tests.cs#L139
Please use `AreEqual`.